mlflow/xgboost/__init__.py

```diff
@@ -583,8 +583,8 @@ def log_feature_importance_plot(features, importance, importance_type):
             # If early_stopping_rounds is present, logging metrics at the best iteration
             # as extra metrics with the max step + 1.
             early_stopping_index = all_arg_names.index("early_stopping_rounds")
-            early_stopping = (
-                num_pos_args >= early_stopping_index + 1 or "early_stopping_rounds" in kwargs
+            early_stopping = num_pos_args >= early_stopping_index + 1 or (
+                "early_stopping_rounds" in kwargs and kwargs["early_stopping_rounds"]
```

Comment:
```suggestion
               in kwargs.get("early_stopping_rounds")
```

Can we use `get` here?